pack/commands.cfg

```diff
@@ -0,0 +1,25 @@
+# -----------------------------------------------------------------
+#
+#      OpenStack standard check
+#      Need  https://github.com/kedark3/check-osp
+#
+# -----------------------------------------------------------------
+
+
+## Check commands for openstack
+define command{
+       command_name     check_vm_count
+       command_line     $PLUGINSDIR$/check_osp_main.py -R $HOSTALIAS$ -u $_HOSTOSP_USER$ -p $_HOSTOSP_PASSWORD$ -t $_HOSTOSP_TENANT$ -m vm_count -c $_HOSTRHV_VM_COUNT_CRIT$ -w $_HOSTRHV_VM_COUNT_WARN$
+}
+define command{
+       command_name     check_locked_disks_count
+       command_line     $PLUGINSDIR$/check_osp_main.py -R $HOSTALIAS$ -u $_HOSTOSP_USER$ -p $_HOSTOSP_PASSWORD$ -t $_HOSTOSP_TENANT$ -m keypair_count -c $_HOSTRHV_LOCKED_DISKS_CRIT$ -w $_HOSTRHV_LOCKED_DISKS_WARN$
+}
+define command{
+       command_name     check_locked_disks_count
```

Comment:
There are two commands called `check_locked_disks_count`, I think this one should be `check_osp_volumes_count`